targets/csharp/make.js

```diff
@@ -92,12 +102,14 @@ function makeDatatypes(apis, sourceDir, apiOutputDir) {
     };
 
     for (var a = 0; a < apis.length; a++) {
-        var modelsLocal = {
-            api: apis[a],
-            makeDatatype: makeDatatype
-        };
+        if (apis[a].calls.length > 0) {
```

Comment:
Nit:
C++ is the only language that optimizes for diving into if statements, rather than trying to bail out early.

Since this is JS, it would be more stylistically appropriate to invert the if, and continue, without tab-shifting the other lines in the for-block.